README.md

```diff
@@ -1,16 +1,19 @@
 # Workflows
-The repo for Workflows that appear within Warp and within [commands.dev](https://www.commands.dev/)
 
+The repo for Workflows that appear within Warp and within [commands.dev](https://www.commands.dev/)
 
 ## What are Workflows?
-Workflows are an easier way to execute and share commands within Warp. They are searchable by name, description, or command and are easily parameterized. 
+
+Workflows are an easier way to execute and share commands within Warp. They are searchable by name, description, or command and are easily parameterized.
 
 ## How Do I Access Workflows?
-Workflows can be accessed within Warp directly within the app, either through the command palette or by pressing `ctrl-shift-r`. 
+
+Workflows can be accessed within Warp directly within the app, either through the Command Palette or by pressing `shift-ctrl-r`.
```

Comment:
why `shift-ctrl-r` instead of `ctrl-shift-r` here?

Reply:
@alokedesai oh you're right my bad